README.rst

```diff
@@ -45,28 +46,29 @@ Library features a Sans-IO core with both asyncio and blocking wrappers:
             print(f"I got some PCAP data {data}")
 
 
-.. |code_ci| image:: https://github.com/PandABlocks/PandABlocks-client/workflows/Code%20CI/badge.svg?branch=master
-    :target: https://github.com/PandABlocks/PandABlocks-client/actions?query=workflow%3A%22Code+CI%22
+.. |code_ci| image:: https://github.com/PandABlocks/PandABlocks-client/actions/workflows/code.yml/badge.svg?branch=master
+    :target: https://github.com/PandABlocks/PandABlocks-client/actions/workflows/code.yml
     :alt: Code CI
 
-.. |docs_ci| image:: https://github.com/PandABlocks/PandABlocks-client/workflows/Docs%20CI/badge.svg?branch=master
-    :target: https://github.com/PandABlocks/PandABlocks-client/actions?query=workflow%3A%22Docs+CI%22
+.. |docs_ci| image:: https://github.com/PandABlocks/PandABlocks-client/actions/workflows/docs.yml/badge.svg?branch=master
+    :target: https://github.com/PandABlocks/PandABlocks-client/actions/workflows/docs.yml
     :alt: Docs CI
 
 .. |coverage| image:: https://codecov.io/gh/PandABlocks/PandABlocks-client/branch/master/graph/badge.svg
     :target: https://codecov.io/gh/PandABlocks/PandABlocks-client
     :alt: Test Coverage
 
-.. |pypi_version| image:: https://badge.fury.io/py/pandablocks.svg
-    :target: https://badge.fury.io/py/pandablocks
+.. |pypi_version| image:: https://img.shields.io/pypi/v/pandablocks.svg
+    :target: https://pypi.org/project/pandablocks
     :alt: Latest PyPI version
 
 .. |license| image:: https://img.shields.io/badge/License-Apache%202.0-blue.svg
     :target: https://opensource.org/licenses/Apache-2.0
     :alt: Apache License
 
+
 ..
-    These definitions are used when viewing README.rst and will be replaced
+    Anything below this line is used when viewing README.rst and will be replaced
     when included in index.rst
 
-See https://pandablocks.github.io/PandABlocks-client for more detailed documentation.
+See https://PandABlocks.github.io/PandABlocks-client for more detailed documentation.
```

Comment:
Looks good

The one thing I can think of is the `github.io` link:

We are using `pandablocks.github.io` instead of `PandaBlocks.github.io`,
though the latter is redirected to the former:
```
$ sphinx-build -b linkcheck docs/
...
(           index: line   11) redirect  https://PandABlocks.github.io/PandABlocks-client - permanently to https://pandablocks.github.io/PandABlocks-client/
...

Might be worth changing this.

Reply:
That's a good point, @M-Chan please can you change `docs_url` to be `"https://{{ github_org | lower }}.github.io/{{ repo_name }}" in the template please?